features/step_definitions/resources.rb

```diff
@@ -125,7 +125,7 @@
 
 # tries to delete resource if it exists and make sure it disappears
 # example: I ensure "hello-openshift" pod is deleted
-Given /^(I|admin) ensures? #{QUOTED} (\w+) is deleted(?: from the#{OPT_QUOTED} project)?( after scenario)?$/ do |by, name, type, project_name, after|
+Given /^(I|admin) ensures? #{QUOTED} (.+) is deleted(?: from the#{OPT_QUOTED} project)?( after scenario)?$/ do |by, name, type, project_name, after|
```

Comment:
you should specify the name with underscores here, not dots. `console_cli_download_console_openshift_io`